src/Behat/Behat/HelperContainer/Exception/HelperContainerException.php

```diff
@@ -11,13 +11,13 @@
 namespace Behat\Behat\HelperContainer\Exception;
 
 use Behat\Testwork\Environment\Exception\EnvironmentException;
-use Interop\Container\Exception\ContainerException;
+use Psr\Container\ContainerExceptionInterface;
 
 /**
  * All HelperContainer exceptions implement this interface.
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
  */
-interface HelperContainerException extends ContainerException, EnvironmentException
+interface HelperContainerException extends ContainerExceptionInterface, EnvironmentException
```

Comment:
to be BC, we should keep extending the interop interface for now (which extends the PSR one). Otherwise, existing try/catch may not work anymore (if they were catching the interop one)